src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.c

```diff
@@ -299,31 +310,31 @@ void func_8089E318(BgMizuMovebg* this, PlayState* play) {
 
     type = MOVEBG_TYPE(this->dyna.actor.params);
     switch (type) {
-        case 3:
-        case 4:
-        case 5:
-        case 6:
+        case MIZUMOVEBG_TYPE_DRAGON_STATUE_BOSS_ROOM:
+        case MIZUMOVEBG_TYPE_DRAGON_STATUE_2:
+        case MIZUMOVEBG_TYPE_DRAGON_STATUE_3:
+        case MIZUMOVEBG_TYPE_DRAGON_STATUE_4:
             if (play->roomCtx.curRoom.num == this->dyna.actor.room) {
                 Matrix_RotateY(BINANG_TO_RAD(this->dyna.actor.world.rot.y), MTXMODE_NEW);
-                Matrix_MultVec3f(&D_8089EBAC, &sp28);
-                this->dyna.actor.child->world.pos.x = this->dyna.actor.world.pos.x + sp28.x;
-                this->dyna.actor.child->world.pos.y = this->dyna.actor.world.pos.y + sp28.y;
-                this->dyna.actor.child->world.pos.z = this->dyna.actor.world.pos.z + sp28.z;
+                Matrix_MultVec3f(&sOffsetPosition2, &offsetPos);
+                this->dyna.actor.child->world.pos.x = this->dyna.actor.world.pos.x + offsetPos.x;
+                this->dyna.actor.child->world.pos.y = this->dyna.actor.world.pos.y + offsetPos.y;
+                this->dyna.actor.child->world.pos.z = this->dyna.actor.world.pos.z + offsetPos.z;
                 this->dyna.actor.child->flags &= ~ACTOR_FLAG_0;
             }
             break;
     }
 }
 
-void func_8089E650(BgMizuMovebg* this, PlayState* play) {
+void BgMizuMovebg_UpdateSmallBlockHookshotTarget(BgMizuMovebg* this, PlayState* play) {
```

Comment:
any reason to not call this `BgMizuMovebg_UpdateHookshotPlatform` like the type `MIZUMOVEBG_TYPE_HOOKSHOT_PLATFORM` ?